diff --git a/src/main/java/com/example/examplemod/quest/QuestManager.java b/src/main/java/com/example/examplemod/quest/QuestManager.java
index 1286f4e8e32dd0264b1c0ab6f383161358e914b3..88c85a6f3469073df46b3d3ee85ddbde79e2c6fe 100644
--- a/src/main/java/com/example/examplemod/quest/QuestManager.java
+++ b/src/main/java/com/example/examplemod/quest/QuestManager.java
@@ -15,50 +15,51 @@ public class QuestManager {
     private static boolean branchCompleted = false;
     private static boolean brushwoodCompleted = false;
     private static boolean startSmithingCompleted = false;
     private static boolean initialFaunaCompleted = false;
     private static boolean startHammersCompleted = false;
     private static boolean ironClusterCompleted = false;
     private static boolean pureIronOreCompleted = false;
     private static boolean firepitCompleted = false;
     private static boolean pyriteCompleted = false;
     private static boolean pyriteFlintCompleted = false;
     private static boolean roughKnivesCompleted = false;
     private static boolean scrapedLeatherCompleted = false;
     private static boolean calcinedIronOreCompleted = false;
     private static boolean calcinedGoldOreCompleted = false;
     private static boolean calcinedTinOreCompleted = false;
     private static boolean brickKilnCompleted = false;
     private static boolean clayPotCompleted = false;
     private static boolean clayCupCompleted = false;
     private static boolean unrefinedTinOreCompleted = false;
     private static boolean unrefinedGoldOreCompleted = false;
     private static boolean cleanedGravelTinOreCompleted = false;
     private static boolean cleanedGravelGoldOreCompleted = false;
     private static boolean spongeMetalsCompleted = false;
     private static boolean cobblestoneAnvilCompleted = false;
     private static boolean metalChunksCompleted = false;
+    private static boolean rawBlanksCompleted = false;
 
 
     public static boolean isPlanksCompleted() {
         return planksCompleted;
     }
 
     public static void setPlanksCompleted(boolean value) {
         planksCompleted = value;
     }
 
 
     public static boolean isSlabsCompleted() {
         return slabsCompleted;
     }
 
     public static void setSlabsCompleted(boolean value) {
         slabsCompleted = value;
     }
 
     public static boolean isCobbleSlabsCompleted() {
         return cobbleSlabsCompleted;
     }
 
     public static void setCobbleSlabsCompleted(boolean value) {
         cobbleSlabsCompleted = value;
@@ -306,50 +307,58 @@ public class QuestManager {
 
     public static boolean isSpongeMetalsCompleted() {
         return spongeMetalsCompleted;
     }
 
     public static void setSpongeMetalsCompleted(boolean value) {
         spongeMetalsCompleted = value;
     }
 
     public static boolean isCobblestoneAnvilCompleted() {
         return cobblestoneAnvilCompleted;
     }
 
     public static void setCobblestoneAnvilCompleted(boolean value) {
         cobblestoneAnvilCompleted = value;
     }
 
     public static boolean isMetalChunksCompleted() {
         return metalChunksCompleted;
     }
 
     public static void setMetalChunksCompleted(boolean value) {
         metalChunksCompleted = value;
     }
 
+    public static boolean isRawBlanksCompleted() {
+        return rawBlanksCompleted;
+    }
+
+    public static void setRawBlanksCompleted(boolean value) {
+        rawBlanksCompleted = value;
+    }
+
     public static boolean isAncientWorldCompleted() {
         return planksCompleted
                 && slabsCompleted
                 && cobbleSlabsCompleted
                 && hewnStonesCompleted
                 && bigBonesCompleted
                 && sharpenedBoneCompleted
                 && stoneToolsCompleted
                 && boneToolsCompleted
                 && combsCompleted
                 && flaxFibersCompleted
                 && branchCompleted
                 && brushwoodCompleted
                 && startSmithingCompleted
                 && initialFaunaCompleted
                 && startHammersCompleted
                 && ironClusterCompleted
                 && pureIronOreCompleted
                 && firepitCompleted
                 && pyriteCompleted
                 && pyriteFlintCompleted
                 && roughKnivesCompleted
                 && scrapedLeatherCompleted
                 && calcinedIronOreCompleted
                 && clayPotCompleted
@@ -370,27 +379,28 @@ public class QuestManager {
         branchCompleted = false;
         brushwoodCompleted = false;
         startSmithingCompleted = false;
         initialFaunaCompleted = false;
         startHammersCompleted = false;
         ironClusterCompleted = false;
         pureIronOreCompleted = false;
         firepitCompleted = false;
         pyriteCompleted = false;
         pyriteFlintCompleted = false;
         roughKnivesCompleted = false;
         scrapedLeatherCompleted = false;
         calcinedIronOreCompleted = false;
         calcinedGoldOreCompleted = false;
         calcinedTinOreCompleted = false;
         brickKilnCompleted = false;
         clayPotCompleted = false;
         clayCupCompleted = false;
         unrefinedTinOreCompleted = false;
         unrefinedGoldOreCompleted = false;
         cleanedGravelTinOreCompleted = false;
         cleanedGravelGoldOreCompleted = false;
         spongeMetalsCompleted = false;
         boneTongsCompleted = false;
         metalChunksCompleted = false;
+        rawBlanksCompleted = false;
     }
 }
