diff --git a/src/main/java/com/example/examplemod/item/BoneTongsItem.java b/src/main/java/com/example/examplemod/item/BoneTongsItem.java
index 196d433cb7fb555d4c26a4a3fd2be8a78765f033..41a8d1ac224d499b12122de4d8a56e79c985de04 100644
--- a/src/main/java/com/example/examplemod/item/BoneTongsItem.java
+++ b/src/main/java/com/example/examplemod/item/BoneTongsItem.java
@@ -5,51 +5,51 @@ import com.example.examplemod.ModContainers;
 import com.example.examplemod.container.BoneTongsContainer;
 import com.example.examplemod.container.EnhancedDualContainer;
 import com.example.examplemod.tileentity.PechugaTileEntity;
 import net.minecraft.block.BlockState;
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.entity.player.PlayerInventory;
 import net.minecraft.entity.player.ServerPlayerEntity;
 import net.minecraft.inventory.container.SimpleNamedContainerProvider;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.ItemUseContext;
 import net.minecraft.nbt.CompoundNBT;
 import net.minecraft.util.ActionResult;
 import net.minecraft.util.ActionResultType;
 import net.minecraft.util.Hand;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.text.TranslationTextComponent;
 import net.minecraft.world.World;
 import net.minecraftforge.common.capabilities.ICapabilityProvider;
 import net.minecraftforge.fml.network.NetworkHooks;
 
 import javax.annotation.Nullable;
 
 public class BoneTongsItem extends Item {
     public BoneTongsItem(Properties properties) {
-        super(properties.durability(20));
+        super(properties.durability(30));
     }
 
     @Nullable
     @Override
     public ICapabilityProvider initCapabilities(ItemStack stack, @Nullable CompoundNBT nbt) {
         return new BoneTongsCapabilityProvider(stack);
     }
 
     @Override
     public ActionResult<ItemStack> use(World world, PlayerEntity player, Hand hand) {
         ItemStack held = player.getItemInHand(hand);
         if (!world.isClientSide && player instanceof ServerPlayerEntity) {
             NetworkHooks.openGui((ServerPlayerEntity) player,
                     new SimpleNamedContainerProvider(
                             (windowId, playerInventory, playerEntity) -> new BoneTongsContainer(windowId, playerInventory, held),
                             new TranslationTextComponent("container.examplemod.bone_tongs")),
                     buffer -> buffer.writeItem(held));
         }
         return ActionResult.success(held);
     }
 
     @Override
     public ActionResultType useOn(ItemUseContext context) {
         World world = context.getLevel();
         if (world.isClientSide) {
