diff --git a/src/main/java/com/example/examplemod/client/screen/main/SystemInfoScreen.java b/src/main/java/com/example/examplemod/client/screen/main/SystemInfoScreen.java
index 68f9c017a2753d0f647257699d786a7eba0cf8d6..5cead3b5e3d30bcd9f9fdaadc2c7867735db6ead 100644
--- a/src/main/java/com/example/examplemod/client/screen/main/SystemInfoScreen.java
+++ b/src/main/java/com/example/examplemod/client/screen/main/SystemInfoScreen.java
@@ -45,45 +45,47 @@ public class SystemInfoScreen extends Screen {
         float ty = (y0 + 30) * inv;
         float lh = (this.font.lineHeight + 2) * inv;
 
         this.font.draw(ms, "Каждый игровой час: +2 жажды и усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "15 минут плавания: +5 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "10 секунд без движения: -10 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "100м бега: +5 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "20 прыжков: +3 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Атака существа: +1 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Бутылка воды: -20 жажды", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Сырая/варёная рыба: +15 жажды", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Кнопка 'Попить': -2 жажды", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Кнопка 'Перемешать': +3 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Заточка камня: +5 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
+        this.font.draw(ms, "Крафт заготовок: +2 усталости", tx, ty, 0xFFFFFF);
+        ty += lh;
         this.font.draw(ms, "Ломание блоков руками: +1 усталости за 5 блоков", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Отдых сидя 10 сек: -15 усталости", tx, ty, 0xFFFFFF);
         ty += lh;
         this.font.draw(ms, "Полный сон: усталость = 0", tx, ty, 0xFFFFFF);
 
         ms.popPose();
 
         super.render(ms, mouseX, mouseY, pt);
     }
 
     @Override
     public void onClose() {
         if (this.minecraft != null && this.minecraft.player != null && this.minecraft.level != null && parent != null) {
             this.minecraft.setScreen(parent);
         } else {
             super.onClose();
         }
     }
-}
\ No newline at end of file
+}
